README.md

```diff
@@ -1,2 +1,145 @@
-# rails-app
-a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Game Time <a name="about-project"></a>
+
+> a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://www.tailwindcss.com/">Tailwind CSS</a></li>
+  </ul>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Signup**
+- **Login**
+- **Add Category**
+- **Add Transactions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 🚀 Live and Video Demo <a name="live and video-demo"></a>
+
+- [ video demo link](https://www.loom.com/share/0175cb699ec040a8bf745978105a47c5)
```

Comment:
[REQUIRED] Kindly add the link to your deployed project here instead of your video presenation